brick/__main__.py

```diff
@@ -101,19 +101,30 @@ def generate_dockerfile_contents(
     secrets=None,
     external_images=None,
     environment=None,
+    chown=None,
 ):
     dockerfile_contents = "# syntax = docker/dockerfile:experimental\n"
     dockerfile_contents += f"FROM {from_image}\n"
+
+    copy_flag_chown = f" --chown={chown}" if chown else ""
+
     if inputs_from_build:
         dockerfile_contents += (
-            "\n".join([f"COPY --from={x[0]} /home/{x[1]} /home/{x[1]}" for x in inputs_from_build])
+            "\n".join(
+                [
+                    f"COPY{copy_flag_chown} --from={x[0]} /home/{x[1]} /home/{x[1]}"
```

Comment:
Nitpick: but would be a bit easier to read this if we added an additional whitespace character here: `COPY {copy_flag_chown}`.

Reply:
In this case the command becomes `COPY  --flag1 --flag2` (with an additional whitespace). Is this what you're suggesting? I didn't want to introduce a double whitespace in order to not mess up with the command line parser, but I'm happy to change it though!

Reply:
Yes, I'm suggesting additional white space for readability. I'm fairly sure most command-line parses would handle multiple white spaces. 

We already have double white space it seems in https://github.com/tmrowco/brick/pull/77/files/8ff3dbc19b52ba67f589012351646cd5470832ea#diff-70115798985cf86a2ba2136857657c87b1888392550f3c723b2a9e02e87c2b95R122